src/Internal.AspNetCore.SiteExtension.Sdk/Internal.AspNetCore.SiteExtension.Sdk.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageId>Internal.AspNetCore.SiteExtension.Sdk</PackageId>
+    <TargetFrameworks>netcoreapp2.0;net46</TargetFrameworks>
```

Comment:
Just double checking to see we want to target netcoreapp2.0 and not 2.1?

Reply:
Actually, this is an internal tool so this makes sense.

Reply:
Right. This is for a task assembly, not a runtime one.

Reply:
Same as in `Internal.AspNetCore.Sdk`